docs/workspaces/create.md

```diff
@@ -31,4 +31,6 @@ To create a workspace:
 10. Optionally, change storage size by clicking **Change**. If you have selected Datasets, Onepanel will automatically adjust storage size based on total size of added Datasets plus an additional 50GB.
 ![](../assets/img/create-195238.png)
 
-11. Click **Create** to create your Workspace. Depending on your machine type, this could take up to 5 minutes.
\ No newline at end of file
+11. Click **Create** to create your Workspace. Depending on your machine type, this could take up to 5 minutes.
+
+[![](../assets/vid/Initiating-a-Workspace.gif)](https://youtu.be/m7wcSFt34Y8)
```

Comment:
@inohmonton99 let's use all lower case for file names and their references in the docs.

Reply:
is the video supposed to also show in YouTube? That link doesn't seem to work. If the video is the same as the gif, then there is no reason to link it.

![image](https://user-images.githubusercontent.com/1211823/69990268-3615cf80-14fa-11ea-888a-f2d48ba83094.png)

Reply:
I believe video links don't work because I've set visibility of these to private, let me try and change it to public so this can be seen

Reply:
video link should be working by now @rushtehrani , please check again

Reply:
> @inohmonton99 let's use all lower case for file names and their references in the docs.

this is noted, will change on next commit, should I proceed and add the other videos I created for jobs/datasets/projects?

Reply:
video, works. Yes, go ahead and make the changes mentioned and add the remaining videos.